protobuf/model_config.proto

```diff
@@ -1377,6 +1408,16 @@ message ModelSequenceBatching
   //@@     model.
   //@@
   repeated ControlInput control_input = 2;
+
+  //@@  .. cpp:var:: State state (repeated)
+  //@@
+  //@@     The implicit state that should be stored in Triton for performing
+  //@@     inference requests in a sequence. Each sequence holds an implicit
+  //@@     state local to itself. The output state provided by the model in 
+  //@@     'output_name' field of the current inference request will be sent 
+  //@@     to the next request in the sequence in the 'input_name' field.
```

Comment:
What will be the state as input of the first request in the sequence?

Reply:
The state will contain garbage data in the first inference request. I'll add a sentence about this.